src/api/Feed.js

```diff
@@ -297,6 +257,7 @@ class Feed extends Base {
         }
 
         this.id = file.id;
+        this.onError = errorCallback;
```

Comment:
In some places we assign this value from the `onError` param and in others from the `errorCallback` param. It's not immediately clear what the distinction is or when to use one or the other.

Reply:
Would adding JSDoc to the `onError` property on the class help? Maybe changing the variable name?

```
     * @param {Function} errorCallback - the error callback which is called after data fetching is complete if there was an error
     * @param {Function} onError - the function to be called immediately after an error occurs
```